PortabilityTools.sln

```diff
@@ -92,7 +92,9 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "backend", "backend", "{0189
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "backend", "backend", "{DE3F5141-1F37-45F6-9A63-697E05E9BF8F}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Functions.Tests", "tests\backend\Functions.Tests\Functions.Tests.csproj", "{CE644844-BE5D-4B95-9C3E-4DEDFCD19C48}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Functions.Tests", "tests\backend\Functions.Tests\Functions.Tests.csproj", "{CE644844-BE5D-4B95-9C3E-4DEDFCD19C48}"
+EndProject
```

Comment:
For reference I found this issue. It seems that 9A19103F-16F7-4668-BE54-9A1E7A4F7556 is the one from latest VS while FAE04EC0-... is a legacy one.  Maybe we should all upgrade our VS to the latest in case old versions revert back to FAE04EC0...

Reply:
[https://github.com/dotnet/project-system/issues/2832](https://github.com/dotnet/project-system/issues/2832)

---
In reply to: [174208896](https://github.com/Microsoft/dotnet-apiport/pull/592#discussion_r174208896) [](ancestors = 174208896)